bugsnag-android-core/src/androidTest/java/com/bugsnag/android/AppDataTest.java

```diff
@@ -52,7 +52,7 @@ public void setUp() throws Exception {
 
     @Test
     public void testAccessors() {
-        assertEquals("com.bugsnag.android.test", appData.get("packageName"));
+        assertEquals("com.bugsnag.android.jvm.test", appData.get("packageName"));
```

Comment:
Is this still correct? There's a number of references to `com.bugsnag.android.jvm` hanging around.

Reply:
Nope, this was missed, I've updated the tests and manifest package ID.